config/runtime/kunit.jinja2

```diff
@@ -67,10 +67,12 @@ cd {src_path} && \
             'parent': parent['_id'],
             'status': 'complete',
             'revision': parent['revision'],
+            'group': parent['group'],
         }
         for node in child_nodes:
             base_node.update({
                 'name': node['name'],
+                'path': parent['path'] + [node['name']],
                 'result': node.get('result'),
```

Comment:
How do we skip the particular test and why?
Need to add logic to set node result to `SKIP`.

Reply:
Err, sorry?  I don't see what you mean.  This PR is about adding the `path` attribute, how is that related to the node result?

Reply:
OK I see.  The "SKIP" result from KUnit is a key in the `RESULT_MAP` dictionary so any test that KUnit has skipped will have a result set to `None` in the data structure.